src/Aquifer.Data/Entities/TranslationPairEntity.cs

```diff
@@ -0,0 +1,28 @@
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+
+namespace Aquifer.Data.Entities;
+
+[EntityTypeConfiguration(typeof(TranslationPairEntityConfiguration))]
+public class TranslationPairEntity
+{
+    public int Id { get; set; }
+    public required LanguageEntity Language { get; set; }
+    public required int LanguageId { get; set; }
+    public required string Key { get; set; }
+    public required string Value { get; set; }
+
+    [SqlDefaultValue("getutcdate()")]
+    public required DateTime Created { get; set; }
+
+    [SqlDefaultValue("getutcdate()")]
+    public required DateTime Updated { get; set; }
+}
+
+public class TranslationPairEntityConfiguration : IEntityTypeConfiguration<TranslationPairEntity>
+{
+    public void Configure(EntityTypeBuilder<TranslationPairEntity> builder)
+    {
+        builder.HasOne(x => x.Language).WithMany(l => l.TranslationPairs).HasForeignKey(tp => tp.LanguageId);
+    }
```

Comment:
Does it work without this piece? I think since we're specifying `Language` above and then have `TranslationPairs` in the `LanguageEntity` class it should be able to calculate this on its own.